content/get-haiku/burn-cd.html

```diff
@@ -48,22 +51,22 @@ <h4>OpenBSD</h4>
 <h4>Mac OS X</h4>
 
 <ul>
- <li><strong>SimplyBurns:</strong> <a href="http://sourceforge.net/projects/simplyburns/">http://sourceforge.net/projects/simplyburns/</a></li>
+ <li><strong>SimplyBurns:</strong> <a href="https://sourceforge.net/projects/simplyburns/">https://sourceforge.net/projects/simplyburns/</a></li>
 </ul>
 
 <h4>Windows</h4>
 
 <ul>
  <li>InfraRecorder (open source): <a href="http://infrarecorder.org" title="InfraRecorder website">http://infrarecorder.org</a></li>
- <li>ImgBurn (freeware): <a href="http://www.imgburn.com" title="ImgBurn website">http://www.imgburn.com</a></li>
- <li>ISO Recorder (freeware): <a href="http://isorecorder.alexfeinman.com/isorecorder.htm" title="ISO Recorder home page">http://isorecorder.alexfeinman.com/isorecorder.htm</a></li>
- <li>Active@ ISO Burner (freeware): <a href="http://www.ntfs.com/iso_burner_free.htm" title="Active@ ISO Burner">http://www.ntfs.com/iso_burner_free.htm</a></li>
+ <li>ImgBurn (freeware): <a href="https://www.imgburn.com" title="ImgBurn website">https://www.imgburn.com</a></li>
+ <li>ISO Recorder (freeware): <a href="https://iso-recorder.en.softonic.com/" title="ISO Recorder home page">https://iso-recorder.en.softonic.com/</a></li>
```

Comment:
Please use the actual website:
http://isorecorder.alexfeinman.com/#/

softonic is some random software archive and I don't know if they can be trusted or if they add random spyware to what they provide.

Reply:
Sorry about that. The original URL didn't work, and I must have messed up when I tried with the domain alone as I got an error. I then just picked a result from a web search...
Anyway, fixed.